pkg/mapper/configmap/configmap.go

```diff
@@ -59,10 +66,12 @@ func (ms *MapStore) startLoadConfigMap(stopCh <-chan struct{}) {
 					FieldSelector: fields.OneTermEqualSelector("metadata.name", "aws-auth").String(),
 				})
 				if err != nil {
-					logrus.Warn("Unable to re-establish watch.  Sleeping for 5 seconds")
+					logrus.Errorf("Unable to re-establish watch: %v, sleeping for 5 seconds.", err)
+					metricsObj.watch.WithLabelValues(metricFailure).Set(metricFailureUnit)
```

Comment:
can you tell more why we need a label of success/failure here? isnt the gauge enough to determine the state?